plugins/woocommerce/changelog/39598-add-order-received-validation-filter

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+Create the filter woocommerce_order_received_verification_required to allow guest users to see the page order-received
```

Comment:
Depending on our thoughts on the other proposals, we could make a further update here to keep things in sync:

```suggestion
Adds filter `woocommerce_order_received_verify_known_shoppers` to allow known shoppers to access the order received page without being logged in.
```